src/main/java/com/twilio/rest/api/v2010/account/IncomingPhoneNumber.java

```diff
@@ -379,6 +388,7 @@ private IncomingPhoneNumber(@JsonProperty("account_sid")
                                 @JsonProperty("voice_method")
                                 final HttpMethod voiceMethod,
                                 @JsonProperty("voice_url")
+                                @JsonDeserialize(using = com.twilio.converter.UriEncodingDeserializer.class)
```

Comment:
This file (and the test file) are generated files (see header at top of files). If we're to make this change in the generator, it would affect all `URI` param types. I _think_ this is fine, but just calling it out here,